samcli/local/common/runtime_template.py

```diff
@@ -96,6 +96,21 @@ def get_local_lambda_images_location(mapping, runtime):
     "java8.al2": ["maven", "gradle"],
 }
 
+RUNTIME_TO_BUILD_IMAGE = {
+    "python3.8": "public.ecr.aws/sam/build-python3.8",
```

Comment:
The name is by design `public.ecr.aws/sam/build-<runtime>`, so it's fine to simply append.

Reply:
this is also used as an allowlist of the runtimes

Reply:
We can still have append to a constant to avoid potential typos. In fact since the name is by design in that format, a set of runtimes and a function to return the URI of one could also make it clear and fool-proof.